Maths/TimeConversion.java

```diff
@@ -0,0 +1,34 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Solution {
+
```

Comment:
Add javadocs comments explaining different parameters. Follow the correct coding guidelines with proper description of the methods. Refer to [DecimalToAnyBase.java](https://github.com/TheAlgorithms/Java/blob/Development/src/main/java/com/conversions/DecimalToAnyBase.java) and [DecimalToAnyBaseTest.java](https://github.com/TheAlgorithms/Java/blob/Development/src/test/java/com/conversions/DecimalToAnyBaseTest.java) for the algorithms and tests coding standards, respectively.